Prog/cosmos special

gammaw opened the pull request:
It's for this takeover: https://leguesswho.com/lineup/independent-community-radio-network-icrn

Approved by prog. 

Please review if possible today for Sat test. 

PS: there is some conflict yes, the new branch's version shall prevail.

tuz666 reviewed (approved):
Looks good!
Ps: only thing which is not clear is why we removed the BoomBal playlist it's still an active show no, we just deleted replacing their slot with the external show, no?

gammaw commented:
> Ps: only thing which is not clear is why we removed the BoomBal playlist it's still an active show no, we just deleted replacing their slot with the external show, no?

Cause it's syntax error if there is a playlist definition if it's never used. Same for weltSchmerz.
